pkg/restclient/volume.go

```diff
@@ -90,9 +90,9 @@ func (c *Client) VolumeStatus(volname string) (api.VolumeStatusResp, error) {
 }
 
 // VolumeStart starts a Gluster Volume
-func (c *Client) VolumeStart(volname string) error {
+func (c *Client) VolumeStart(volname string, req api.VolumeStartReq) error {
```

Comment:
Second argument can be `force bool`, request can be formed inside this func.

Reply:
@aravindavk done.